spec/features/tslr_claim_spec.rb

```diff
@@ -159,4 +159,116 @@
     visit claim_path("qts-year")
     expect(page).to have_current_path(root_path)
   end
+
+  context "editing a response" do
+    let(:claim_school) { create(:school, :tslr_eligible, name: "Claim School") }
+    let(:current_school) { create(:school, :tslr_eligible, name: "Current School") }
+
+    let(:claim) do
+      claim = create(:tslr_claim,
+        :eligible_and_submittable,
+        complete: true,
+        employment_status: :different_school,
+        claim_school: claim_school)
+      claim.current_school = current_school
+      claim.save
+      claim
+    end
+
+    before do
+      allow_any_instance_of(ClaimsController).to receive(:current_claim) { claim }
+      visit claim_path("check-your-answers")
+    end
+
+    scenario "Teacher can edit a field" do
+      find("a[href='#{claim_path("national-insurance-number")}']").click
+
+      fill_in "National Insurance number", with: "AB123456C"
+      click_on "Continue"
+
+      expect(claim.reload.national_insurance_number).to eq("AB123456C")
```

Comment:
Could we either explicitly set or check the existing value, so we know this is actually a change?